domains/ayskobtww.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "aysko-discord",
+        "email": "marckelyfc@icloud.com"
+    },
+    "record": {
+        "URL": "guns.lol/aysko"
```

URL records need to have protocol
```suggestion
        "URL": "https://guns.lol/aysko"
```